SEC/Equities/EquityInstruments.rdf

```diff
@@ -537,6 +612,26 @@
 		<skos:definition>right that give the shareholder higher priority with respect to certain pre-defined dividend payments over other shareholders and that pay out first in the case of liquidation</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;PreferredDividend">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-eq-eq;Dividend"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasFixedDividendRate"/>
```

Comment:
But a preferred dividend might be variable surely.

Reply:
Most of the time it is fixed rate.  This restriction is optional, however, and there are two subclasses - fixed and adjustable.